README.md

```diff
@@ -1,17 +1,39 @@
 [![Build](https://github.com/TUD-OS/latex-template/actions/workflows/build.yml/badge.svg)](https://github.com/TUD-OS/latex-template/actions/workflows/build.yml)
 
-Was ist das?
-============
+## Was ist das?
 
 Dieses Diplomarbeits-Latex-Template stellt ein Skelett für eine
 Diplomarbeit mit zugehöriger Make-Datei zur Verfügung.
 
-Checkout Submodules First
-===================
+## Checkout Submodules First
 In order to use the `$ make checkbiw` script please execute: `git submodule update --init --recursive`
 
-Build Prerequisites
-===================
+## So geht's los
+
+Als erstes solltest du in diplom.tex alle Vorkommen von Otto
+Mustermann, "Dein Titel" und "Dein Betreuer" ersetzen.
+
+`diplom.tex` will deine Aufgabe als PDF einbinden. Es sucht
+`images/diplom-aufgabe.pdf`, was eine A4 Seite sein muss. Mit
+
+    convert <dein-gescanntes-bild> diplom-aufgabe.pdf
+
+solltest du eine beliebige Bilddatei in ein PDF umwandeln können, wenn
+ImageMagick auf deinem System installiert ist.
+
+An diesen Punkt sollte `make` ein `diplom.pdf` produzieren.
+
+Das Template unterstützt sowohl englischen und deutschen Text. Englisch ist
+standardmäßig eingestellt. Für deutschen Text kann der letzte `\selectlanguage`
+Aufruf in `diplom.tex` einfach weggelassen werden.
+
+## How To Build
```

Comment:
Markdown typically has a newline between headings and content, and content and code blocks. This is also what you did above.